<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>437</width>
                <height>533</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Battery Control</string>
        </property>
        <property name="dockNestingEnabled">
            <bool>false</bool>
        </property>
        <widget class="QWidget" name="centralwidget">
            <property name="enabled">
                <bool>true</bool>
            </property>
            <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                </sizepolicy>
            </property>
            <property name="autoFillBackground">
                <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout">
                <item row="1" column="1">
                    <widget class="QTableWidget" name="table_batteries">
                        <property name="enabled">
                            <bool>true</bool>
                        </property>
                        <property name="editTriggers">
                            <set>QAbstractItemView::NoEditTriggers</set>
                        </property>
                        <property name="selectionMode">
                            <enum>QAbstractItemView::SingleSelection</enum>
                        </property>
                        <property name="selectionBehavior">
                            <enum>QAbstractItemView::SelectRows</enum>
                        </property>
                        <property name="columnCount">
                            <number>2</number>
                        </property>
                        <attribute name="horizontalHeaderVisible">
                            <bool>true</bool>
                        </attribute>
                        <attribute name="horizontalHeaderCascadingSectionResizes">
                            <bool>false</bool>
                        </attribute>
                        <attribute name="horizontalHeaderDefaultSectionSize">
                            <number>60</number>
                        </attribute>
                        <attribute name="horizontalHeaderStretchLastSection">
                            <bool>true</bool>
                        </attribute>
                        <attribute name="verticalHeaderVisible">
                            <bool>false</bool>
                        </attribute>
                        <attribute name="verticalHeaderDefaultSectionSize">
                            <number>21</number>
                        </attribute>
                        <column>
                            <property name="text">
                                <string>Battery ID</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>Voltage</string>
                            </property>
                        </column>
                    </widget>
                </item>
                <item row="1" column="0">
                    <layout class="QVBoxLayout" name="verticalLayout">
                        <property name="sizeConstraint">
                            <enum>QLayout::SetFixedSize</enum>
                        </property>
                        <item>
                            <widget class="QPushButton" name="button_start">
                                <property name="text">
                                    <string>Start</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="button_stop">
                                <property name="text">
                                    <string>Stop</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="verticalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                    </layout>
                </item>
                <item row="2" column="0" colspan="2">
                    <widget class="QTableWidget" name="table_dives">
                        <property name="editTriggers">
                            <set>QAbstractItemView::NoEditTriggers</set>
                        </property>
                        <attribute name="horizontalHeaderStretchLastSection">
                            <bool>true</bool>
                        </attribute>
                        <attribute name="verticalHeaderVisible">
                            <bool>false</bool>
                        </attribute>
                        <attribute name="verticalHeaderDefaultSectionSize">
                            <number>21</number>
                        </attribute>
                        <column>
                            <property name="text">
                                <string>Dive ID</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>Drone</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>Batteries</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>Time</string>
                            </property>
                        </column>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>437</width>
                    <height>22</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
